CHANGELOG.md

```diff
@@ -1,3 +1,436 @@
+}
+```
+
+new:
+
+```hcl
+component "contour" {
+  .
+  .
+}
+
+```
+
+##### Prometheus-operator component
+
+* Prometheus specific variables are now under a `prometheus` block.
+* Variable `external_url` is now removed and now configured under
+    `prometheus.ingress.host`. Remove the http scheme when configuring.
+* A new optional `prometheus.ingress` sub-block is introduced to expose
+Prometheus over ingress.
+
+old:
+
+```hcl
+component "prometheus-operator" {
+  .
+  .
+  prometheus_metrics_retention = "14d"
+  prometheus_external_url      = "https://api.example.com/prometheus"
+  prometheus_storage_size      = "50GiB"
+  prometheus_node_selector = {
+    "kubernetes.io/hostname" = "worker3"
+  }
+  .
+  .
+}
+```
+
+new:
+
+```hcl
+component "prometheus-operator" {
+  .
+  .
+  prometheus {
+    metrics_retention = "14d"
+    storage_size      = "50GiB"
+    node_selector = {
+      "kubernetes.io/hostname" = "worker3"
+    }
+
+    ingress {
+      host                       = "prometheus.example.lokomotive-k8s.net"
+      class                      = "contour"
+      certmanager_cluster_issuer = "letsencrypt-production"
+    }
+    .
+    .
+  }
+  .
+  .
+}
+```
+
+Check out the new syntax in the [Prometheus Operator configuration
+reference](https://github.com/kinvolk/lokomotive/blob/v0.2.0/docs/configuration-reference/components/prometheus-operator.md)
+for details.
+
+##### Upgrade steps
+
+Go to your cluster's directory and run the following command.
+
+```
+lokoctl cluster apply
+```
+
+The update process typically takes about 10 minutes.
+After the update, running `lokoctl health` should result in an output similar to the following.
+
+```
+Node                     Ready    Reason          Message
+
+lokomotive-controller-0  True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-0    True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-1    True     KubeletReady    kubelet is posting ready status
+lokomotive-1-worker-2    True     KubeletReady    kubelet is posting ready status
+Name      Status    Message              Error
+
+etcd-0    True      {"health":"true"}
+```
+
+##### Post upgrade steps
+
+##### Openebs
+
+OpenEBS control plane components and data plane components work independently.
+Even after the OpenEBS Control Plane components have been upgraded to 1.11.0,
+the Storage Pools and Volumes (both jiva and cStor) will continue to work with
+older versions.
+
+>Upgrade functionality is still under active development. It is highly
+>recommended to schedule a downtime for the application using the OpenEBS PV
+>while performing this upgrade. Also, make sure you have taken a backup of the
+>data before starting the below upgrade procedure. - [Openebs
+documentation](https://github.com/openebs/openebs/blob/master/k8s/upgrades/README.md#step-3-upgrade-the-openebs-pools-and-volumes)
+
+
+###### Upgrade cStor Pools
+
+* Extract the SPC name using `kubectl get spc`:
```

Comment:
We should perhaps open an issue to automate that.

Reply:
yes, I've already added to the todo list.

Reply:
https://github.com/kinvolk/lokomotive/issues/744